lua/ulx/modules/sh/cfc_owoify.lua

```diff
@@ -0,0 +1,44 @@
+local CATEGORY_TYPE = "Chat"
+local OWOIFYDICT = { ove = "uv", na = "nya", ne = "nye", ni = "nyi", no = "nyo", nu = "nyu", r = "w", l = "w", O = "owo", U = "uwu", wh = "w" }
+local owoifyTargets = {}
+
+local function owoifyString( inputStr )
+    for k, v in pairs( OWOIFYDICT ) do
+        inputStr = string.Replace( inputStr, k, v )
+    end
+
+    return inputStr
+end
+
+local function onPlayerSay( ply, inputStr )
+    if not table.HasValue( owoifyTargets, ply ) then return end
+
+    return owoifyString( inputStr )
+end
+
+local function owoifyOn( caller, targets )
+    table.Add( owoifyTargets, targets )
+    hook.Add( "PlayerSay", "CFC_ULX_OwoifyString", onPlayerSay )
+    ulx.fancyLogAdmin( caller, "#A owoified #T", targets )
+end
+
+local function owoifyOff( caller, targets )
+    for k, v in pairs( owoifyTargets ) do
+        owoifyTargets[k] = nil
+    end
+
+    if table.Count( owoifyTargets ) == 0 then
+        hook.Remove( "PlayerSay", "CFC_ULX_OwoifyString" )
+    end
+
+    ulx.fancyLogAdmin( caller, "#A unowoified #T", targets )
+end
+
+local owoifyCommand = ulx.command( CATEGORY_TYPE, "ulx owoify", owoifyOn, "!owoify" )
+owoifyCommand:defaultAccess( ULib.ACCESS_ADMIN )
+owoifyCommand:help( "Owoifies target(s) so they are unable to chat normally. (say: !owoify) (opposite: ulx unowoify)" )
+owoifyCommand:addParam{ type = ULib.cmds.PlayersArg }
+
+local unOwoifyCommand = ulx.command( CATEGORY_TYPE, "ulx unowoify", owoifyOff, "!unowoify" )
+unOwoifyCommand:defaultAccess( ULib.ACCESS_ADMIN )
+unOwoifyCommand:addParam{ type = ULib.cmds.PlayersArg }
```

Comment:
I believe ULX has a concept of "opposite" commands. You might want to look at the other examples to see if we do that anywhere. You could also look through the ULX code (or docs if they exist?) and see if you could find something about it.

Reply:
yeah, it used to have a help line for both but I copied what ulx gimp does

Reply:
If i'm not mistaken there is no help line for ulx ungimp, just the info for gimp and the info after it in that line

Reply:
theres a function gimp uses 
`gimp:setOpposite`
https://github.com/TeamUlysses/ulx/blob/master/lua/ulx/modules/sh/chat.lua#L190

I believe that's what phatso was referring to, and probably what you should for this command